foundationsync/indico_foundationsync/blueprint.py

```diff
@@ -0,0 +1,30 @@
+# This file is part of the CERN Indico plugins.
+# Copyright (C) 2014 - 2019 CERN
+#
+# The CERN Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License; see
+# the LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from flask import redirect
+from werkzeug.exceptions import NotFound
+
+from indico.core.plugins import IndicoPluginBlueprint
+from indico.modules.rb.models.rooms import Room
+
+
+blueprint = IndicoPluginBlueprint('foundationsync', 'indico_foundationsync')
```

Comment:
`__name__` for the second arg? at least we do that in pretty much all other plugins..